### PR TITLE
[KIECLOUD-735] - Prepare RHPAM repositories for 7.13.5 release

### PR DESCRIPTION
**Pull request by spolti**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: kieserver/modules/kieserver/module.yaml [hunk 1]
```diff
@@ -28,17 +27,17 @@ envs:
...
   md5: "adb6f8394caa54c444cf0571d2a8debb"
 - name: "rhpam_business_central_distribution.zip"
-  # rhpam-7.13.4-business-central-eap7-deployable.zip
+  # rhpam-7.13.5-business-central-eap7-deployable.zip
   md5: "87c555d71825b88d3576fe6fba3a998c"
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.30.redhat-00005.jar
   md5: "44e25fa5b93454bc9cfc005f8cb98b53"
-- name: "kie-server-services-jbpm-cluster-7.67.0.Final-redhat-00024.jar"
+- name: "kie-server-services-jbpm-cluster-7.13.5.Final-redhat-00024.jar"
```
Comment (rgdoliveira) [inline kieserver/modules/kieserver/module.yaml]: @spolti changes in jbpm-event-emitters-kafka and kie-server-services-jbpm-cluster versions seems wrong

**Review by rgdoliveira — CHANGES_REQUESTED** [review]

**Review by desmax74 — DISMISSED** [review]
Ok, but @rgdoliveira is asking a double check on some jar version

File: kieserver/modules/kieserver/module.yaml (hunk 1 above)
Comment (spolti) [reply kieserver/modules/kieserver/module.yaml]: it is the script version the regex catches this as well.

Comment (spolti) [inline kieserver/modules/kieserver/module.yaml]:
```suggestion
- name: "kie-server-services-jbpm-cluster-7.67.0.Final-redhat-00024.jar"
```

File: kieserver/modules/kieserver/module.yaml [hunk 2]
```diff
@@ -28,17 +27,17 @@ envs:
...
-  # rhpam-7.13.4-business-central-eap7-deployable.zip
+  # rhpam-7.13.5-business-central-eap7-deployable.zip
   md5: "87c555d71825b88d3576fe6fba3a998c"
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.30.redhat-00005.jar
   md5: "44e25fa5b93454bc9cfc005f8cb98b53"
 - name: "kie-server-services-jbpm-cluster-7.67.0.Final-redhat-00024.jar"
   md5: "8385a5ef35efd5f7c3849ce3cfd5ac9a"
-- name: "jbpm-event-emitters-kafka-7.67.0.Final-redhat-00024.jar"
+- name: "jbpm-event-emitters-kafka-7.13.5.Final-redhat-00024.jar"
```
Comment (spolti) [inline kieserver/modules/kieserver/module.yaml]:
```suggestion
- name: "jbpm-event-emitters-kafka-7.67.0Final-redhat-00024.jar"
```

File: kieserver/modules/kieserver/module.yaml [hunk 3]
```diff
@@ -28,17 +27,17 @@ envs:
...
-  # rhpam-7.13.4-business-central-eap7-deployable.zip
+  # rhpam-7.13.5-business-central-eap7-deployable.zip
   md5: "87c555d71825b88d3576fe6fba3a998c"
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.30.redhat-00005.jar
   md5: "44e25fa5b93454bc9cfc005f8cb98b53"
 - name: "kie-server-services-jbpm-cluster-7.67.0.Final-redhat-00024.jar"
   md5: "8385a5ef35efd5f7c3849ce3cfd5ac9a"
-- name: "jbpm-event-emitters-kafka-7.67.0.Final-redhat-00024.jar"
+- name: "jbpm-event-emitters-kafka-7.67.0Final-redhat-00024.jar"
```
Comment (spolti) [inline kieserver/modules/kieserver/module.yaml]:
```suggestion
- name: "jbpm-event-emitters-kafka-7.67.0.Final-redhat-00024.jar"
```

**Review by rgdoliveira — APPROVED** [review]
approved, LGTM

**Pull request merged**